jobs/build-podman.Jenkinsfile

```diff
@@ -0,0 +1,379 @@
+def gitref, commit, shortcommit
+def containername = 'podman-fcos'
```

Comment:
This is only used in the `lock()`; seems OK to drop this variable and hardcode "build-podman-fcos" in the `lock()`?